test/integration/utils.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs-extra')
+const os = require('os')
+const path = require('path')
+
+const rootDirectory = fs.realpathSync(process.cwd())
+const resolvePath = relativePath => path.resolve(rootDirectory, relativePath)
+
+let stepCount = 1
+
+function printStepTitle(title) {
```

Comment:
How about `makeStepCounter` and no global `stepCount`?

Reply:
Could you please explain what do you have in mind?